pubspec.yaml

```diff
@@ -4,8 +4,8 @@ homepage: https://github.com/Workiva/workiva_analysis_options
 description: Workiva's shared static analysis options.
 
 environment:
-  sdk: ">=2.4.0 <3.0.0"
+  sdk: '>=2.12.0 <3.0.0'
```

Comment:
Are you thinking it's safe to bump the minimum because Workiva devs will be using 2.12 before we need to update this package again?